test/e2e/certificates_test.go

```diff
@@ -174,7 +177,7 @@ var _ = Describe("ACME Certificate", Ordered, func() {
 	})
 
 	Context("dns-01 challenge using ambient credentials", func() {
-		It("should obtain a valid LetsEncrypt certificate in AWS mint/passthrough cluster", func() {
+		It("should obtain a valid LetsEncrypt certificate using ClusterIssuer on AWS mint/passthrough cluster", func() {
```

Comment:
Given we are adding more scenarios, let's retitle so that even glance at title can get the exact checkpoint.
```suggestion
		It("should obtain a valid LetsEncrypt certificate using ambient ClusterIssuer on AWS mint/passthrough cluster", func() {
```

Reply:
Will update

Reply:
About `ambient`/`explicit`, is already mentioned here:

https://github.com/openshift/cert-manager-operator/blob/207d9062d78f16b786998a7989d2487eee77bf21/test/e2e/certificates_test.go#L176

https://github.com/openshift/cert-manager-operator/blob/207d9062d78f16b786998a7989d2487eee77bf21/test/e2e/certificates_test.go#L65